packages/wallets/src/features/cashier/components/WalletCashierHeader/WalletCashierHeader.tsx

```diff
@@ -1,84 +1,162 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
 import classNames from 'classnames';
 import { useHistory, useLocation } from 'react-router-dom';
 import { useActiveWalletAccount } from '@deriv/api';
-import { WalletCardIcon } from '../../../../components/WalletCardIcon';
-import { WalletGradientBackground } from '../../../../components/WalletGradientBackground';
+import { WalletCardIcon, WalletGradientBackground, WalletText } from '../../../../components';
 import { WalletListCardBadge } from '../../../../components/WalletListCardBadge';
 import useDevice from '../../../../hooks/useDevice';
 import CloseIcon from '../../../../public/images/close-icon.svg';
+import IcCashierAdd from '../../../../public/images/ic-cashier-deposit.svg';
+import IcCashierStatement from '../../../../public/images/ic-cashier-statement.svg';
+import IcCashierTransfer from '../../../../public/images/ic-cashier-transfer.svg';
+import IcCashierWithdrawal from '../../../../public/images/ic-cashier-withdrawal.svg';
+import ResetBalance from '../../../../public/images/plus-thin.svg';
 import './WalletCashierHeader.scss';
 
-const realAccountTabs = ['deposit', 'withdraw', 'transfer', 'transactions'];
-const virtualAccountTabs = ['withdraw', 'transfer', 'transactions', 'reset-balance'];
+type TProps = {
+    hideWalletDetails: boolean;
+};
+
+const realAccountTabs = [
+    {
+        icon: <IcCashierAdd />,
```

Comment:
Can we rename this to IcCashierDeposit instead?

Reply:
@nijil-deriv sure